### ISSUE TITLE
install.sh is broken

### ISSUE THREAD

**Issue by lischenko** [Phoenix1747/RouteryPi#2]
The script breaks when run with `sh install.sh` under recent raspbian (mine was 2018-06-27-raspbian-stretch-lite). 
```
install.sh: 4: install.sh: function: not found
An error occured! Aborting....
```
The reason is most likely because `/bin/sh` is pointing to `dash` now: https://wiki.ubuntu.com/DashAsBinSh

An easy fix for that is to use posix syntax for the function: replace `function check_errors {` with `check_errors() {`

When run with `bash install.sh` it passes further but breaks when trying to write the `/etc/network/interfaces` file:

```
Writing /etc/network/interfaces...
'
bash: line 1: auto: command not found
bash: line 2: iface: command not found
bash: line 5: auto: command not found
...
```

**Comment by NuclearPhoenixx** [comment]
Thanks, I'll look into it.

**Comment by fabioassuncao** [comment]
@lischenko manual installation is most recommended at this time, and some adjustments can be made immediately.

**Comment by NuclearPhoenixx** [comment]
Should be fixed with commit cf7d781. Please let me know if there are any more issues!
